x/stakeibc/types/message_add_validator.go

```diff
@@ -50,5 +51,21 @@ func (msg *MsgAddValidator) ValidateBasic() error {
 	if err := utils.ValidateWhitelistedAddress(msg.Creator); err != nil {
 		return err
 	}
+	// name validation
+	if len(msg.Name) == 0 {
+		return sdkerrors.Wrapf(sdkerrors.ErrInvalidRequest, "name is required")
+	}
+	// commission validation
+	if msg.Commission > 100 {
```

Comment:
is this guaranteed to be greater than 0 since it's an unsigned int? And if so, wouldn't the same apply to the weight validation below and maxclaims validation [here](https://github.com/Stride-Labs/stride/pull/71/files#diff-80eae880f9395eddcac85b86fc08e5f55f9f4b651ffd80ae9faf2ff775124f8fR47)